MediaBrowser.Providers/MediaInfo/FFProbeProvider.cs

```diff
@@ -61,8 +61,8 @@ public class FFProbeProvider : ICustomMetadataProvider<Episode>,
             NamingOptions namingOptions)
         {
             _logger = logger;
-            _audioResolver = new AudioResolver(localization, mediaEncoder, namingOptions);
-            _subtitleResolver = new SubtitleResolver(BaseItem.LocalizationManager);
+            _audioResolver = new MediaInfoResolver(localization, mediaEncoder, namingOptions, DlnaProfileType.Audio);
+            _subtitleResolver = new MediaInfoResolver(localization, mediaEncoder, namingOptions, DlnaProfileType.Subtitle);
```

Comment:
Not sure this is the way to go, now you can't use these with DI.

Maybe `MediaInfoResolver` should be an abstract class with an abstract property `Type` which gets inherited by `AudioResolver` and `SubtitleResolver`?